dor/adapters/eventpublisher.py

```diff
@@ -18,18 +18,42 @@
 channel = conn.channel()
 
 channel.queue_declare('fileset')
-channel.queue_declare('packaging')
-channel.queue_declare('ingest')
+channel.queue_declare('packaging.work')
+channel.queue_declare('ingest.work')
+
+channel.exchange_declare(exchange="packaging", exchange_type="fanout")
 
 router = {
-    commands.CreatePackage: 'packaging',
-    events.PackageGenerated: 'packaging'
+    commands.CreatePackage: 'packaging.work',
+    commands.IngestPackage: 'ingest.work'
 }
 
-def publish(message: Message) -> None:
-    key = router.get(type(message))
+def publish(command: commands.Command) -> None:
```

Comment:
I think we will probably want the external messages to just be sent out, and let the routing happen outside of the sending context. As a metaphor, it's outgoing mail but we don't know the address (at the place where the command or event is raised).

However, I am thinking about whether that really makes sense for commands. Usually, for an imperative action, the recipient is known. I'm looking for examples to help shape our early opinions.